CMakeLists.txt

```diff
@@ -191,18 +191,12 @@ unset(check_pie_output)
 # The core_interface library aims to encapsulate common build flags.
 # It is intended to be a usage requirement for all other targets.
```

Comment:
I think this documentation should have been updated? `core_interface` is a requirement for all targets now?

Reply:
Please see https://github.com/hebasto/bitcoin/pull/256.